google/gax/__init__.py

```diff
@@ -33,7 +33,7 @@
 import collections
 
 
-__version__ = '0.12.5'
+__version__ = '0.12.6'
```

Comment:
Done.

Good catch -- this is definitely breaking, since we no longer support the beta entry point to the gRPC stub, which is required by older gRPC autogen.